_overviews/contributors/index.md

```diff
@@ -285,72 +248,62 @@ uid                  Julien Richard-Foy <julien.richard-foy@epfl.ch>
+ 2. Create a new Secret containing the base64 encoded secret of your private key which you can obtain
+ by running: 
+```
+# macOS
+gpg --armor --export-secret-keys $LONG_ID | base64
+# Ubuntu (assuming GNU base64)
+gpg --armor --export-secret-keys $LONG_ID | base64 -w0
+# Arch
+gpg --armor --export-secret-keys $LONG_ID | base64 | sed -z 's;\n;;g'
+# FreeBSD (assuming BSD base64)
+gpg --armor --export-secret-keys $LONG_ID | base64
+# Windows
+gpg --armor --export-secret-keys %LONG_ID% | openssl base64
+```
+ 3. Publish your public key signature to a public server, for example [http://keyserver.ubuntu.com:11371](http://keyserver.ubuntu.com:11371/).
+ You can obtain the signature by running:
+```
+# macOS and linux
+gpg --armor --export $LONG_ID
+# Windows
+gpg --armor --export %LONG_ID%
+```
 (Replace `(key ID)` with **your** key ID)
-
-Add the `ci/pubring.asc` file (which contains your public key) to your repository. The `secring.asc` file
-(which contains your private key) should **not** be added as it is to the repository, so make sure it will
-be ignored by Git by adding it to the `.gitignore` file:
-
-~~~
-ci/secring.asc
-~~~
-
-Encrypt it with the `travis` tool:
-
-~~~ bash
-$ travis encrypt-file ci/secring.asc ci/secring.asc.enc --add
-~~~
-
-As advised in the command output, make sure to add the `secring.asc.enc` to the git repository.
-
-The `--add` option above adds a line like the following to your `.travis.yml` file:
-
-~~~ diff
-before_install:
-  - openssl aes-256-cbc -K $encrypted_602f530300eb_key -iv $encrypted_602f530300eb_iv -in ci/secring.asc.enc -out ci/secring.asc -d
-~~~
-
-Finally, add export your PGP passphrase to the `.travis.yml` file:
-
-~~~ 
-$ travis encrypt PGP_PASSPHRASE="(your passphrase)" --add
-~~~
-
-(Replace `(your passphrase)` with your actual passphrase)
-
+ 
+ 
 #### Publish From the CI Server
 
-On Travis CI, you can define a
-[conditional stage](https://docs.travis-ci.com/user/build-stages/#specifying-stage-order-and-conditions)
-publishing the library when a tag is pushed:
+On GitHub Actions, you can define a workflow to publish the library when a tag is pushed:
 
 ~~~ yaml
-jobs:
-  include:
-    - stage: test
-      script: sbt test
-    - stage: deploy
-      if: tag =~ ^v
-      script: sbt publishSigned sonatypeRelease
-~~~
+# .github/workflows/release.yml
+name: Continuous publication
+on:
+  push:
+    tags:
 
-The last step is to tell your build definition how to retrieve the PGP passphrase from the `PGP_PASSPHRASE`
-environment variable and to use the `pubring.asc` and `secring.asc` files as the PGP key pair.
-Include the following settings in your `build.sbt` file:
-
-~~~ scala
-pgpPublicRing := file("ci/pubring.asc")
-pgpSecretRing := file("ci/secring.asc")
-pgpPassphrase := sys.env.get("PGP_PASSPHRASE").map(_.toArray)
-~~~
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
I’m not sure this will work because `actions/checkout@v2` does not fetch all the repository’s history. In particular, it does not fetch Git tags, which are necessary for `sbt-dynver` (used by `sbt-ci-release`) to set up the release version.

I’ve been using [this](https://github.com/endpoints4s/endpoints4s/blob/4f03ed0a15353bea8a332757851c4918a9d6b4f4/.github/workflows/ci.yml#L55-L56) or [that](https://github.com/julienrf/play-json-derived-codecs/blob/aa0c78a008e088dbd77768e04a3e633b48f8f24e/.github/workflows/release.yml#L14) as workarounds, I wonder if there is a more established practice.

Reply:
Good catch! I prefer [the first option](https://github.com/endpoints4s/endpoints4s/blob/4f03ed0a15353bea8a332757851c4918a9d6b4f4/.github/workflows/ci.yml#L55-L56)